lib/rex/post/meterpreter/extensions/stdapi/tlv.rb

```diff
@@ -47,6 +47,10 @@ module Stdapi
 TLV_TYPE_SEARCH_GLOB        = TLV_META_TYPE_STRING  | 1231
 TLV_TYPE_SEARCH_ROOT        = TLV_META_TYPE_STRING  | 1232
 TLV_TYPE_SEARCH_RESULTS     = TLV_META_TYPE_GROUP   | 1233
+TLV_TYPE_SEARCH_DATE        = TLV_META_TYPE_RAW     | 1235
```

Comment:
Based on this value being placed right into the hash, shouldn't this be `TLV_META_TYPE_STRING`?

Reply:
I've adjusted the code so that both search query dates and the resultant mtime date are sent as UINTs (unix epoch).